src/utils/jsHelper/helper.tsx

```
@@ -342,97 +360,97 @@ export function getSubmmitedDoc(userData, document) {
-      console.warn(`Invalid range format in IncomeRange: ${range.value}`);
-      continue;
-    }
-    
-    if (income >= min && income <= max) {
-      return range.value;
-    }
-  }
-
-  return "";
+	const income = parseFloat(annualIncome);
+	if (isNaN(income)) return '';
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)